### DISCUSSION TITLE
Tumblr likes only first page will download!

### DISCUSSION THREAD

**Discussion by Wolfendog** [TumblThreeApp/TumblThree#564] [Q&A (Question & Answer)] [answered]
Hello!

I hope I ain't missing something, but whenever I try to download my likes on tumblr (I am authenticated), TumblThree will
only download the first page and nothing more! I tired with and without a starting date. Is anyone else having this problem?
I tried completely deleting everything and re-downloading TumblThree. I can't find a solution.

Thank you in advance if anyone knows what's wrong.

cheers

**Comment by thomas694** [comment]
There are fixes in the new version. Please try it.

**Reply by Wolfendog** [reply]
I've deleted the files in "C:\Users\myname\AppData\Local\TumblThree" as well as the downloaded files of the old version. 

I then tried to open the new version, but the window never shows up and in the task manager the TumblThree.exe keeps using CPU and the RAM usage is slowly rising. I've been waiting a few minutes, but the window never comes. RAM usage was already over 1GB. In AppData no new file is created, so I don't have a log to look at. I'm on Windows 11, tried installing the newest updates as well.

I also tried downloading the previous version, just to see if that would open, it does. But as soon as I try the new version, the stuff above happens.

**Reply by jacketzz** [reply]
> I then tried to open the new version, but the window never shows up and in the task manager the TumblThree.exe keeps using CPU and the RAM usage is slowly rising. I've been waiting a few minutes, but the window never comes. RAM usage was already over 1GB. In AppData no new file is created, so I don't have a log to look at. I'm on Windows 11, tried installing the newest updates as well.

Having the same issue, also on Windows 11!

**Reply by thomas694** [reply, marked as answer]
The problem wasn't only on Windows 11. Anyway, you should update to 2.15.1.

**Reply by Wolfendog** [reply]
Thank you so much, now it works again! 😁
